bones/bone.py

```diff
@@ -409,15 +413,19 @@ def serialize(self, skel: 'SkeletonInstance', name: str, parentIndexed: bool) ->
 					res.append(self.singleValueSerialize(singleValue, skel, name, parentIndexed))
 			else:  # No Languages, not Multiple
 				res = self.singleValueSerialize(newVal, skel, name, parentIndexed)
-			skel.dbEntity[name] = res
-			# Ensure our indexed flag is up2date
-			indexed = self.indexed and parentIndexed
-			if indexed and name in skel.dbEntity.exclude_from_indexes:
-				skel.dbEntity.exclude_from_indexes.discard(name)
-			elif not indexed and name not in skel.dbEntity.exclude_from_indexes:
-				skel.dbEntity.exclude_from_indexes.add(name)
-			return True
-		return False
+		else:
+			res = None
+
+		skel.dbEntity[name] = res
```

Comment:
It maybe is better to just leave `skel.dbEntity[name]` unset in case the value is none, what do you think @tsteinruecken